src/GlimpseCore.Client/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We should be launching the client alongside a sample project which will be triggered through Visual Studio. Sharing this launch configuration is probably unneeded and may vary between developers.